client/my-sites/plan-features-2023-grid/billing-timeframe.tsx

```diff
@@ -3,54 +3,67 @@ import {
 	isWpcomEnterpriseGridPlan,
 	PLAN_BIENNIAL_PERIOD,
 	PLAN_ANNUAL_PERIOD,
+	PlanSlug,
+	getPlanSlugForTermVariant,
+	TERM_ANNUALLY,
 } from '@automattic/calypso-products';
 import { formatCurrency } from '@automattic/format-currency';
 import { localize, TranslateResult, useTranslate } from 'i18n-calypso';
 import { FunctionComponent } from 'react';
 import { useSelector } from 'react-redux';
+import usePlanPrices from 'calypso/my-sites/plans/hooks/use-plan-prices';
 import { getCurrentUserCurrencyCode } from 'calypso/state/currency-code/selectors';
 
 interface Props {
 	planName: string;
 	billingTimeframe: TranslateResult;
 	billingPeriod: number | null | undefined;
-	rawPrice: number | null;
-	maybeDiscountedFullTermPrice: number | null;
-	annualPricePerMonth: number | null;
 	isMonthlyPlan: boolean;
 }
 
 function usePerMonthDescription( {
-	rawPrice,
-	maybeDiscountedFullTermPrice,
-	annualPricePerMonth,
 	isMonthlyPlan,
 	planName,
 	billingPeriod,
 }: Omit< Props, 'billingTimeframe' > ) {
 	const translate = useTranslate();
 	const currencyCode = useSelector( getCurrentUserCurrencyCode );
+	const planPrices = usePlanPrices( {
+		planSlug: planName as PlanSlug,
+		returnMonthly: isMonthlyPlan,
+	} );
+	const planYearlyVariantPrices = usePlanPrices( {
```

Comment:
I found this is bit hard to read on L51. What do you think about renaming this to:
```suggestion
	const planYearlyVariantPricesPerMonth = usePlanPrices( {
```

Reply:
Sounds good! Updated.